ophyd/areadetector/plugins.py

```diff
@@ -105,15 +108,16 @@ def enable_on_stage(self):
 
         a convenience method for adding ('enable', 1) to stage_sigs
         """
-        self.stage_sigs['enable'] = 1
+        self.enable_on_stage = True
```

Comment:
Is this a typo? It looks to me like this method just overrides itself with `True`.

Reply:
I think I might have forgotten an underscore